modules/cnf-measuring-latency.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/cnf-performing-platform-verification-latency-tests.adoc
+
+:_content-type: CONCEPT
+[id="cnf-measuring-latency_{context}"]
+= Measuring latency
+
+The `cnf-tests` image uses the `oslat` tool to measure the latency of the system.
+
+oslat:: Behaves similarly to a CPU-intensive DPDK application and measures all the interruptions and disruptions to the busy loop that simulates CPU heavy data processing.
+
+The tests introduce the following environment variables:
+
+.Latency test environment variables
+[cols="1,3", options="header"]
+|====
+|Environment variables
+|Description
+
```

Comment:
LATENCY_TEST_RUN is missing here as well.
see https://github.com/openshift/openshift-docs/pull/53084#discussion_r1030118151

Reply:
Will apply this in a separate PR to all versions: https://github.com/openshift/openshift-docs/pull/53162

Reply:
Addressed in https://github.com/openshift/openshift-docs/pull/53162